src/main/java/world/bentobox/bentobox/database/objects/Island.java

```diff
@@ -1148,6 +1154,20 @@ public void setRankCommand(String command, int rank) {
         this.commandRanks.put(command, rank);
     }
 
+    /**
+     * @return the reserved
```

Comment:
```suggestion
     * Returns whether this Island is currently reserved or not.
     * If {@code true}, this means no blocks, except a bedrock one at the center of the island, exist.
     * @return {@code true} if this Island is reserved, {@code false} otherwise.
     * @since 1.6.0
```